plugins/destination/sqlite/client/migrate.go

```diff
@@ -131,14 +130,14 @@ func (*Client) canAutoMigrate(changes []schema.TableColumnChange) bool {
 }
 
 // This is the responsibility of the CLI of the client to lock before running migration
-func (c *Client) Migrate(ctx context.Context, tables schema.Tables) error {
+func (c *Client) migrate(_ context.Context, force bool, tables schema.Tables) error {
```

Comment:
`ctx` should be utilized in the calls to database (so `c.createTableIfNotExist` should accept ctx, etc).
You should switch to [`sql.DB.ExecContext`](https://pkg.go.dev/database/sql#DB.ExecContext) call for proper `ctx` utilization